web/themes/custom/genlyd_aarhus/templates/block/itk-floating-help-block.html.twig

```diff
@@ -30,7 +30,8 @@
     "phoneLabel": floating_help_phone_label,
     "phone": floating_help_phone,
     "emailLabel": floating_help_email_label,
-    "email": floating_help_email
+    "email": floating_help_email,
+    "emailTitle": "Send email to " ~ floating_help_email | trans
```

Comment:
This is not the correct way to translate... It will result in a new translation string for each floating_help_email... So every time it is change it will have to be translated again.